src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -262,28 +237,11 @@ class ContentSidebar extends PureComponent<Props, State> {
             return defaultView;
```

Comment:
What will happen if the default view is set to something like skills but there are no skills on the file? We can address that in a later change.

Reply:
oh right, sidebar is not GA'ed nor documented, so I'll remove it then